ravedude/Cargo.toml

```diff
@@ -14,7 +14,7 @@ categories = ["embedded", "hardware-support", "development-tools"]
 colored = "2.0.0"
 tempfile = "3.2.0"
 serialport = "4.0.0"
-anyhow = "1.0.38"
+anyhow = "=1.0.38"
```

Comment:
This shouldn't be neccessary.  Are you sure you're building `ravedude` with stable instead of the old nightly that is pinned for the AVR stuff?  You might need to use

```shell
cargo +stable build
```

Reply:
Hm, maybe we should add a `rust-toolchain.toml` to the `ravedude/` directory as well, to override its toolchain to `stable`?

Reply:
Not sure if adding another rust-toolchain is a good idea. People might want to use different versions of the compiler for whatever reason, that would be unnecessarily strict.

Reply:
Otherwise agree that pinning the dep doesn't make sense, didn't occur to me to use different compiler version for ravedude.

Reply:
I'm most worried about people falling into this trap again.  If we can solve it for 80% of cases, that already means a lot less traffic in the issue tracker & other help channels.  For the rest, you can still overwrite the rust-toolchain with your own preferences anyway, `rustup` provides [plenty of mechanisms](https://rust-lang.github.io/rustup/overrides.html) to do that.

But let's keep this out of the trinket-pro PR, I can take care of that separately.